src/dash-table/dash/Sanitizer.ts

```diff
@@ -99,7 +100,16 @@ export default class Sanitizer {
             headerFormat = ExportHeaders.Ids;
         }
 
+        const active_cell = props.cell_selectable ?
+            props.active_cell :
+            undefined;
+
+        const selected_cells = props.cell_selectable ?
+            props.selected_cells :
+            defaultProps.selected_cells
```

Comment:
Make sure `active_cell` and `selected_cells` have default values if not selectable.